### PR TITLE
Taking a swing at the range input

### PR DESCRIPTION
**Pull request by jakearchibald**
I'm very jetlagged so maybe this is simpler for very bad reasons.

### CODE REVIEW

File: src/components/Output/custom-els/RangeInput/index.ts [hunk 1]
```diff
@@ -89,60 +46,66 @@ class RangeInputElement extends HTMLElement {
...
+    get() {
+      return this._input[prop];
+    },
+    set(val) {
+      this._input[prop] = val;
+      this._reflectAttributes();
+      this._update();
+    },
+  });
+}
```
Comment (jakearchibald) [inline src/components/Output/custom-els/RangeInput/index.ts]: There's some type mess here, but it feels pretty well contained, and doesn't lose types from the outside (like cleanMerge does)

Comment (developit) [reply src/components/Output/custom-els/RangeInput/index.ts]: I'd take type mess over the verbosity I ended up having to avoid it.

**Comment by jakearchibald** [comment]
I've gone ahead and implemented start/end. @developit are you happy merging this into your branch (after review of course)?

File: src/components/Output/custom-els/RangeInput/index.ts [hunk 2]
```diff
@@ -88,61 +47,118 @@ class RangeInputElement extends HTMLElement {
...
 
+  private _startGtEnd(): boolean {
+    const startNum = parseFloat(this.start);
+    const endNum = parseFloat(this.end);
+    return startNum > endNum;
+  }
+
   private _reflectAttributes() {
-    for (const attributeName in REFLECTED_ATTRIBUTES) {
+    if (this.name === 'near_lossless') debugger;
```
Comment (surma) [inline src/components/Output/custom-els/RangeInput/index.ts]: Remove? ^^

**Comment by jakearchibald** [comment]
@surma @developit ok, I ditched the start/end attributes and replaced them with value flipping in Preact. Seems to work much better.

**Comment by jakearchibald** [comment]
Shut up @googlebot

**Pull request merged**